### PR TITLE
Issue 125 - As a user I want a delay before seeing the filter applied to my image, so I can get in position on the footprints in time

### PR DESCRIPTION
**Pull request by isaac-lapworth**
w/ UX designs

# Checklist:

Related work items: #125 

Preview url: [TODO | N/A]

- [ ] Tests have been added for new functionality
- [ ] Work has been reviewed by other team members
- [ ] Work has been reviewed by @craigayre or @eclennett

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,99 +1,100 @@
 {
```
Comment (eclennett) [inline package.json]: why has all the indentation changed in this file...

File: functions/package-lock.json [hunk 2]
```diff
@@ -2675,6 +2675,11 @@
       "resolved": "https://registry.npmjs.org/negotiator/-/negotiator-0.6.1.tgz",
       "integrity": "sha1-KzJxhOiZIQEXeyhWP7XnECrNDKk="
     },
+    "node-fetch": {
```
Comment (eclennett) [inline functions/package-lock.json]: i presume this is a dependency of "install-fucntions"

File: src/components/SnackChat/SnackChat.js [hunk 3]
```diff
@@ -48,24 +56,101 @@ class SnackChat extends Component {
...
+  };
+
+  playGettingInPositionAnimation = () => {
+    const numberOfFallingSnacks = 3;
+    const itemPositions = [];
+    for (let i = 1; i <= numberOfFallingSnacks; i++) {
+      itemPositions.push({
+        x: (this.canvas.current.width * i) / (numberOfFallingSnacks + 1),
+        y:
+          Math.random() * 1.4 * this.canvas.current.height -
```
Comment (eclennett) [inline src/components/SnackChat/SnackChat.js]: we dont know what 1.4 and 0.2 actually are so perhaps we can have them as names variables

File: src/components/SnackChat/SnackChat.js [hunk 4]
```diff
@@ -48,24 +56,101 @@ class SnackChat extends Component {
...
+          0.2 * this.canvas.current.height,
+        rotation: Math.random() * 2 * Math.PI
+      });
+    }
+
+    const gettingInPositionAnimation = () => {
```
Comment (eclennett) [inline src/components/SnackChat/SnackChat.js]: is this really a good name
what about "set...."
that is what you are doing afte rall

File: src/components/SnackChat/SnackChat.js [hunk 5]
```diff
@@ -156,11 +246,12 @@ class SnackChat extends Component {
 
     // Video background
     this.ctx.save();
-    this.ctx.scale(-1, 1);
+    this.ctx.scale(-1.6, 1.6);
```
Comment (eclennett) [inline src/components/SnackChat/SnackChat.js]: these 1.6 seem to be flighing around a lot
variable for these too if you think?

Comment (eclennett) [reply src/components/SnackChat/SnackChat.js]: you have video background up the top
do you need both? function?

File: src/components/SnackChat/SnackChat.js [hunk 6]
```diff
@@ -191,35 +282,72 @@ class SnackChat extends Component {
...
+              Get ready!
+            </div>
+          </header>
+        )}
         <div className="snackchat-body">
           <canvas ref={this.canvas} width={FEED_SIZE} height={FEED_SIZE} />
         </div>
-        <div style={{display: 'none'}}>
+        <div
+          style={{
```
Comment (eclennett) [inline src/components/SnackChat/SnackChat.js]: you didnt actually change anything here
why has the indentation changed

File: package.json (hunk 1 above)
Comment (isaac-lapworth) [reply package.json]: I edited the file to add the `install-functions` script, I'm assuming that prettier converts tabs to spaces for `.json` files

File: src/components/SnackChat/SnackChat.scss [hunk 7]
```diff
@@ -14,3 +16,152 @@
...
 }
+
+.snackchat--header-text {
+  padding-top: 100px;
+  font-size: 35px;
+  display: inline-block;
+  vertical-align: top;
+  position: absolute;
+  &-left {
+    left: 2%;
```
[the remark this answers is not in the document]
Comment (isaac-lapworth) [reply src/components/SnackChat/SnackChat.scss]: Originally tried with padding/margins but this ended up being much simpler

File: src/components/Home/Home.js [hunk 8]
```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import './Home.css';
-import HomeHandsSlot from './assets/HandsSlot.svg';
-import HomeHandsRight from './assets/HandsRight.svg';
-import HomeHandsCenter from './assets/HandsCenter.svg';
-import HomeHandsLeft from './assets/HandsLeft.svg';
+import HomeHandsSlot from './../../utils/assets/hands/HandsSlot.svg';
+import HomeHandsRight from './../../utils/assets/hands/HandsRight.svg';
+import HomeHandsCenter from './../../utils/assets/hands/HandsCenter.svg';
+import HomeHandsLeft from './../../utils/assets/hands/HandsLeft.svg';
```
Comment (cainy393) [inline src/components/Home/Home.js]: With the SVGs being statis files would it not make sense to move them to /public along with the fonts?

Comment (cainy393) [reply src/components/Home/Home.js]: I think that would be logical personally.

Comment (isaac-lapworth) [reply src/components/Home/Home.js]: Will leave this for another PR

**Pull request merged**